pkg/observer/controller.go

```diff
@@ -309,6 +370,15 @@ func (c *controller) Run() {
 	klog.V(3).Infof("waiting for %v to allow changes to settle", c.WaitInterval)
 	select {
 	case <-time.After(c.WaitInterval):
+		timeout := 0
+		retries := 20
+		for c.safeToStartCycle() == false {
+			time.Sleep(10 * time.Second)
+			timeout++
+			if timeout >= retries {
+				klog.Fatalf("Not safe to start CNR. %v retries exceeded", retries)
```

Comment:
Does this crash the observer?

Reply:
Yes. If the cluster is actively scaling up/down it might be safer to crash and wait a new cycle of the observer

Reply:
I'd prefer to wait for the new cycle of observer without the crash. Just change the control flow rather than crash the container. Thoughts?